src/configuration/services/pagespeed.md

```diff
@@ -0,0 +1,41 @@
+# PageSpeed Module
+
+PageSpeed is a family of 4 tools developped by Google Inc to optimize of websites performance.
+
+Pagespeed module, also known as mod_pagespeed, is an open source HTTP module for Apache or Nginx, which runs several optimizations on the resources of your website, on the server side.
+
+See the [PageSpeed docs](https://developers.google.com/speed/) for more information.
+
+## Supported versions
+
+* 1.12
+
+## Setup
+
+In your ``.platform/services.yaml``:
+
+```yaml
+pagespeed:
+    type: pagespeed:1.12-rc
+    configuration:
+        endpoints:
+            app: {}
+    relationships:
+        app: "myapp:http"
+```
+
+The pagespeed proxy exposes an endpoint app and this endpoint uses the upstream relationship named app.
```

Comment:
Technically it's not `app`, it's whatever your app container is named. `app` is just what most of the examples use.

This page should probably explain, at least a little, about how we're sticking an interstitial service between the router and the app.  That's a weird concept that we don't have anywhere else yet. 😄

Reply:
@Crell We haven't documented that, even in draft, for Varnish, yet?